_posts/2015-11-15-sinatra-app.markdown

```diff
@@ -139,18 +226,40 @@ By the time our application is complete, it should look similar to the folder st
 
 <img src="/images/sinatra_visual_tree.png">
 
-## *5.* Setup Coding
+### *2b.* Commit the skeleton to git
+
+If you wish to commit this skeleton folder structure to git, you may notice that the empty folders we created aren't being committed; git will only remember files, not folders. To make git remember our structure, you must create a file inside each empty folder. The convention is to create a file called `.keep`. The dot at the start of the name means that your computers operating system should ignore them.
+
+{% highlight sh %}
+touch app/.keep
+touch app/controllers/.keep
+touch app/models/.keep
+touch app/views/.keep
+touch db/.keep
+touch files/.keep
+
+git add .
+git commit -m "folder structure"
+git push origin master
+{% endhighlight %}
+
+## *3.* Configure the App
 
 We are now ready to begin developing our application.
 
+### *3b.* Sinatra
+
 The first step is to set up the `application.rb` file. Double click on the file to open it (you will find it under the `config` folder). Take some time to understand what we are about to put into this file.
-Note you must save all files in cloud9 (`cmd + s` or menu options `file -> save`)
+Note you must save all files in cloud9 (`command ⌘ + s`(mac), `control + s`(windows and linux) or menu options `File -> Save`)
 
 {% highlight sh %}
+# require gems and classes
```

Comment:
the syntax is pretty cryptic in this file to keep it nice and short, so comments are kinda required for learning purposes